src/pivot-table/components/helpers/__tests__/get-scrollable-areas-dimensions.test.ts

```diff
@@ -0,0 +1,134 @@
+import type { Rect } from "../../../../types/types";
+import getScrollableAreasDimensions from "../get-scrollable-areas-dimensions";
+
+describe("getScrollableAreasDimensions()", () => {
+  let tableRect: Rect;
+  let containerHeight: number;
+  let leftGridHeight: number;
+  let topGridHeight: number;
+  let dataGridHeight: number;
+  let allRowsVisible: boolean;
+
+  let totalWidth: number;
+  let leftGridWidth: number;
+  let rightGridWidth: number;
+  let verticalScrollbarWidth: number;
+  let horizontalScrollbarHeight: number;
+
+  beforeEach(() => {
+    tableRect = { width: 200, height: 340 };
+    containerHeight = 340;
+    leftGridHeight = 300;
+    topGridHeight = 40;
+    dataGridHeight = 300;
+    allRowsVisible = false;
+
+    totalWidth = 200;
+    leftGridWidth = 80;
+    rightGridWidth = 120;
+    verticalScrollbarWidth = 13;
+    horizontalScrollbarHeight = 13;
+  });
+
+  const callFunction = () =>
+    getScrollableAreasDimensions({
+      tableRect,
+      containerHeight,
+      leftGridHeight,
+      topGridHeight,
+      dataGridHeight,
+      allRowsVisible,
+      totalWidth,
+      leftGridWidth,
+      rightGridWidth,
+      verticalScrollbarWidth,
+      horizontalScrollbarHeight,
+    });
+
+  test("check result object structure to be correct", () => {
```

Comment:
This seems redundent since you other tests are already checking all possible properties